tests/e2e/test_arxiv_harvest.py

```diff
@@ -228,3 +197,56 @@ def _all_in_status(status):
         interaction_name='ticket_new',
         times=1,
     )
+
+
+@with_mitmproxy
+def test_harvest_nucl_th_and_jlab_curation(inspire_client, mitm_client):
+    inspire_client.e2e.schedule_crawl(
+        spider='arXiv_single',
+        workflow='article',
+        url='http://export.arxiv.org/oai2',
+        identifier='oai:arXiv.org:1806.05669',  # nucl-th record
+    )
+
+    halted_entry = wait_for(lambda: _all_in_status(inspire_client, 'COMPLETED'))
+
+    entry = inspire_client.holdingpen.get_detail_entry(halted_entry.workflow_id)
+
+    assert entry.arxiv_eprint == '1806.05669'
+    assert entry.control_number is 42
+    assert entry.title == 'Probing the in-Medium QCD Force by Open Heavy-Flavor Observables'
+
+    # check literature record is available and consistent
+    record = inspire_client.literature.get_record(entry.control_number)
+    assert record.title == entry.title
+
+    # check that the external services were actually called
+    mitm_client.assert_interaction_used(
+        service_name='LegacyService',
+        interaction_name='robotupload',
+        times=1,
+    )
+    mitm_client.assert_interaction_used(
+        service_name='RTService',
+        interaction_name='ticket_new',
+        times=1,
+    )
+
+    def _get_ticket_content():
+        curr_path = os.path.dirname(__file__)
+        ticket_file = os.path.join(curr_path, 'scenarios', 'harvest_nucl_th_and_jlab_curation', 'RTService', 'ticket_new.yaml')
+        content = yaml.load(open(ticket_file))
+        ticket_content = urllib2.unquote(content['request']['body']).decode('utf8')
+        return re.search('\/workflows\/edit_article\/[0-9]+', ticket_content).group(0)
+
+    curation_link = _get_ticket_content()
+    assert inspire_client._client.get(curation_link).status_code == 200
+
+    new_entries = wait_for(lambda: n_hp_entries(inspire_client, 2))
+    assert len(new_entries) == 2
+    edit_article_wf = filter(lambda entry: entry.status == 'WAITING', new_entries)[0]
+
+    entry = inspire_client.holdingpen.get_detail_entry(edit_article_wf.workflow_id)
+    inspire_client.holdingpen.resume_wf(entry)
```

Comment:
we should check that it went in and the changes were made to the actual record (check also the robotupload at the end)